src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CanonicalXmlElement.cs

```diff
@@ -30,12 +30,12 @@ public void Write(StringBuilder strBuilder, DocPosition docPos, AncestralNamespa
             SortedList nsListToRender = new SortedList(new NamespaceSortOrder());
             SortedList attrListToRender = new SortedList(new AttributeSortOrder());
 
-            XmlAttributeCollection attrList = Attributes;
+            XmlAttributeCollection? attrList = Attributes;
             if (attrList != null)
             {
                 foreach (XmlAttribute attr in attrList)
                 {
-                    if (((CanonicalXmlAttribute)attr).IsInNodeSet || Utils.IsNamespaceNode(attr) || Utils.IsXmlNamespaceNode(attr))
+                    if ((((CanonicalXmlAttribute)attr)).IsInNodeSet || Utils.IsNamespaceNode(attr) || Utils.IsXmlNamespaceNode(attr))
```

Comment:
Why the extra parens?

Reply:
Well spotted - I can only think it's a result of fat-fingering